### ISSUE TITLE
Please Support command line arguments to specify the directory and filename every captured image

### ISSUE THREAD

**Issue by saccohuo** [ShareX/ShareX#2030]
I have been using plain text tool (called org mode) to note, so I need the captured image to be in  the same directory of the note file. I want to call ShareX with command line argument so that I can specify the location and filename of every captured image.
But I cannot find some options in the "Task settings" and available command line argument for this purpose. The only option that I find is the "Save As" option in the "Task Settings", which cannot meet my requirement.

So is there any method to implement this or is there any chance that you will provide the command line argument for this requirement?

**Comment by Jaex** [comment]
Is where you gonna save images changes frequently?

**Comment by saccohuo** [comment]
Yeah, I wanna set the specified path every time I save the image with command line.
Pass the path to save as the argument of the command, then ShareX can save it there.

**Comment by Jaex** [comment]
How you can give file name when its image encoding can change depending on image size.

**Comment by saccohuo** [comment]
You mean that the extension name of the capture image will depend on its size? I didn't know this feature. 
If it does, I think I will pass the filename except the extension name as argument, and ShareX will decide which extension name to be used.
